### DISCUSSION TITLE
faire la distinction entre acheté et trouvé

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#743] [Nouvelles idées]
Quand on obtient un item, il faudrait que le titre de l'embed dépende de si on l'a trouvé ou acheté

![image](https://user-images.githubusercontent.com/60353007/143765321-73271525-a284-4b77-a5e1-4edfbed8348e.png)

**Comment by BastLast** [comment]
pourquoi ?

**Comment by BastLast** [comment]
c'est chiant à dev je vois pas ce que ca apporte

**Reply by Pietagorh** [reply]
ça apporte de la logique
"Il vous propose d'acheter un truc"
"Ok"
"Vous avez trouvé quelque chose !"

**Reply by BastLast** [reply]
¯\_(ツ)_/¯

**Comment by BastLast** [comment]
+ ca devrait etre une suggestion, c'est pas un bug.

**Reply by Pietagorh** [reply]
Effectivement

**Comment by BastLast** [comment]
je converti en discussion
